tests/model_eval_context_test.go

```diff
@@ -0,0 +1,22 @@
+package test
```

Comment:
I didn't put this `_test.go` file under root directory together with their source `*.go` files because I want to simulate the package behavior from a third party's view.